2010-02-09  Aurelien Jarno  <aurelien@aurel32.net>

	* sysdeps/unix/sysv/linux/arm/nptl/bits/atomic.h: remove useless
	include.

---
 ports/sysdeps/unix/sysv/linux/arm/nptl/bits/atomic.h |    1 -
 1 file changed, 1 deletion(-)

--- a/ports/sysdeps/unix/sysv/linux/arm/nptl/bits/atomic.h
+++ b/ports/sysdeps/unix/sysv/linux/arm/nptl/bits/atomic.h
@@ -16,7 +16,6 @@
    <http://www.gnu.org/licenses/>.  */
 
 #include <stdint.h>
-#include <sysdep.h>
 
 
 typedef int8_t atomic8_t;
